search_movies/urls.py

```diff
@@ -0,0 +1,22 @@
+"""search_movies URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.10/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import include, url
+from django.contrib import admin
+
+urlpatterns = [
+    url(r'^movies/', include('imdb.urls')),
```

Comment:
when we type the URL in the browser on our localhost it search for the given matches in the urls.py in the project and from there it passs the flow to the urls.py in the app .. we include this  urls.py of the app in the urls.py of the project so the control flows from there to the urls.py in the app and further tries to match the url from the regular expression in the urls.py of the app . 

include function is used  to tell to which file we need to transfer the flow and check for the further urlpattern